From eda104a3862ca3bf3325938648d49c29e958e90b Mon Sep 17 00:00:00 2001
From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sat, 9 May 2015 17:47:52 +0200
Subject: [PATCH] can: headers: make header files self contained

This patch add the missing #include-s to the dev.h and led.h, so that they
don't rely on the C-file that's includem them, to include further header files.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 include/linux/can/dev.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/linux/can/dev.h b/include/linux/can/dev.h
index 309d2aa..02c615e 100644
--- a/include/linux/can/dev.h
+++ b/include/linux/can/dev.h
@@ -14,8 +14,9 @@
 #define CAN_DEV_H
 
 #include <linux/can.h>
-#include <linux/can/netlink.h>
 #include <linux/can/error.h>
+#include <linux/can/netlink.h>
+#include <linux/netdevice.h>
 
 /*
  * CAN mode
-- 
1.8.3.1

